pkg/appgw/backendhttpsettings.go

```diff
@@ -23,43 +23,67 @@ const (
 	DefaultConnDrainTimeoutInSec = 30
 )
 
-func newBackendIds(ingressList []*v1beta1.Ingress) map[backendIdentifier]interface{} {
+func newBackendIds(ingressList []*v1beta1.Ingress, serviceList []*v1.Service) map[backendIdentifier]interface{} {
 	backendIDs := make(map[backendIdentifier]interface{})
 	for _, ingress := range ingressList {
 		if ingress.Spec.Backend != nil {
-			glog.Infof("Ingress spec has no backend. Adding a default.")
-			backendIDs[generateBackendID(ingress, nil, nil, ingress.Spec.Backend)] = nil
+			backendID := generateBackendID(ingress, nil, nil, ingress.Spec.Backend)
+			glog.Info("Ingress spec has no backend. Adding default backend:", backendID.serviceKey(), backendID.serviceFullName())
```

Comment:
Info message needs correction.
`ingress.Spec.Backend` gives us the default backend in a case where the `ingress.Spec.Rules[].Path[].Backend` is not specified for a particular rule.

Reply:
Good catch! Thanks!